packages/sync/README.md

```diff
@@ -17,3 +17,140 @@ To enable debug please execute in console
 Some certain features can be enabled separately
 
 `localStorage.debug = 'AeroGearSync:OfflineMutations*'`
+
+# Importing the package
+```javascript
+import {
+  createClient,
+  strategies
+} from '@aerogear/datasync-js';
+```
+
+# Configuration
+To provide custom configuration to the client, the following options are available. If you wish, these are also available by using the `DataSyncConfig` interface from the SDK.
+
+```javascript
+declare var window: any;
+
+let config: DataSyncConfig = {
+  httpUrl: "my-custom-url",
+  wsUrl: "my-custom-ws-url",
+  dataIdFromObject: "id",
+  storage: window.localStorage,
+  conflictStrategy: strategies.diffMergeClientWins,
+  customLinkBuilder: customLinkBuilder,
+  networkStatus: customNetworkStatus,
+  mutationsQueueName: "offline-mutations-queue",
+  squashing: true
+}
+```
+
+Config | Description | Default
+------ | ------ | ------
+httpUrl | The URL of your http server | N/A
+wsUrl | The URL of your websocket | N/A
+dataIdFromObject | The name of the field to be used as ID | "id"
```

Comment:
This is not longer used. I will create ticket to restore that to not forget about it